src/main/java/Sidtacphi.java

```diff
@@ -0,0 +1,255 @@
+import java.util.Objects;
+import java.util.Scanner;
+import java.util.ArrayList;
+
+/**
+ * Sidtacphi is the main class for the Sidtacphi bot.
+ */
+public class Sidtacphi {
+    private static ArrayList<Task> taskList = new ArrayList<>();
+    enum TaskType {
+        TODO,
+        EVENT,
+        DEADLINE
+    }
+    
+    /**
+     * The main method for the Sidtacphi class.
+     * 
+     * @param args
+     */
+    public static void main(String[] args) {
+        startBot();
+    }
+
+    /**
+     * This method starts the Sidtacphi bot.
+     */
+    private static void startBot() {
+        String logo = " _______  ___   ______   _______  _______  _______  _______  __   __  ___  \n"
+                + "|       ||   | |      | |       ||   _   ||       ||       ||  | |  ||   | \n"
+                + "|  _____||   | |  _    ||_     _||  |_|  ||       ||    _  ||  |_|  ||   | \n"
+                + "| |_____ |   | | | |   |  |   |  |       ||       ||   |_| ||       ||   | \n"
+                + "|_____  ||   | | |_|   |  |   |  |       ||      _||    ___||       ||   | \n"
+                + " _____| ||   | |       |  |   |  |   _   ||     |_ |   |    |   _   ||   | \n"
+                + "|_______||___| |______|   |___|  |__| |__||_______||___|    |__| |__||___| \n";
+        System.out.println("____________________________________________________________");
+        System.out.println("Hello from\n" + logo + "\n");
+        System.out.println("What can I do for you?");
+        System.out.println("____________________________________________________________");
+
+        readInputs();
+    }
+
+    /**
+     * This method stops the Sidtacphi bot.
+     */
+    private static void stopBot() {
+        System.out.println("\nSidtacphi: Goodbye non-Euclidean life form.");
+        System.out.println("\n____________________________________________________________");
+    }
+
+    private static int tryParseInt(String text, int defaultVal) {
+        try {
+          return Integer.parseInt(text);
+        } catch (NumberFormatException e) {
+          return defaultVal;
+        }
+    }
+
+    /**
+     * The method for reading inputs for the bot.
```

Comment:
Similarly to the previous comment, perhaps you might want to change the description to "Reads inputs for the bot."